src/components/graph-view.js

```diff
@@ -1490,18 +1494,26 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
             renderDefs={renderDefs}
           />
           <g className="view" ref={el => (this.view = el)}>
-            {this.renderBackground()}
+            <Background
+              gridSize={gridSize}
+              backgroundFillId={backgroundFillId}
+              renderBackground={renderBackground}
+            />
 
             <g className="entities" ref={el => (this.entities = el)} />
           </g>
         </svg>
-        <div className="graph-controls-wrapper" />
+        <div
+          id="react-digraph-graph-controls-wrapper"
```

Comment:
I added an ID for the graph controls wrapper to speed up graph control rendering.